go-controller/pkg/node/default_node_network_controller_test.go

```diff
@@ -315,8 +315,7 @@ var _ = Describe("Node", func() {
 				_, err = config.InitConfig(ctx, fexec, nil)
 				Expect(err).NotTo(HaveOccurred())
 				config.Default.EncapPort = encapPort
-
-				err = setEncapPort()
+				err = setEncapPort(ctx.Context)
```

Comment:
do we need to re-use the context here ? Wouldn't it be better to just use a freshly spawned context `context.Background()` ?

Reply:
sure @maiqueb , it makes sense. done.